packages/babel-parser/test/helpers/run-fixture-tests.js

```diff
@@ -100,6 +140,22 @@ function runParseTest(parse, test, onlyCompareErrors) {
   }
 
   const actual = parseWithRecovery(parse, source, filename, options);
+
+  if (ignoreLocations) {
+    if (
+      actual.ast &&
+      expected.ast &&
+      !actual.ast.errors &&
+      !options.plugins?.includes("estree") &&
+      !options.ranges
+    ) {
+      removeLocations(actual.ast);
+      removeLocations(expected.ast);
+    } else {
+      return;
+    }
+  }
+
   const difference = new Difference(
```

Comment:
Thanks @liuxingbaoyu for working on this test!

Below in the file, when we are fuzzing and using random line/col/index, `test.original` will be false and thus the `if (!test.original) return` will suppress any error. Should we replace it with
```js
  if (!test.original) {
    const err = new Error();
    err.message = `Test Failed: ${testLocation}\nFixtureError.fromDifference: ${
      FixtureError.fromDifference(difference, actual).message
    }`;
    throw err;
  }
```